#! /bin/sh -e
## 25_install_into_datadir by Jens Peter Secher <jps@debian.org>
## Prepared by Sebastian Schaffert

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0
--- sml-mode-3.9.5.orig/Makefile
+++ sml-mode-3.9.5/Makefile
@@ -43,8 +43,8 @@
 elibdir = $(lispdir)/elib
 
 # the directory where you install the info doc
-infodir = $(prefix)/info
-docdir = $(prefix)/doc
+infodir = $(datadir)/info
+docdir = $(datadir)/doc
 
 EMACS	= emacs
 MAKEINFO= makeinfo
@@ -134,7 +134,7 @@
 	$(MKDIR) $(docdir)
 	$(CP) `find . -type f -name '*.dvi' -print` $(docdir)/
 
-install: install_elc install_info install_startup install_el
+install: install_elc install_info install_el
 
 clean:
 	$(RM) *~ core .\#* $(TEXEXTS)
